### ISSUE TITLE
Error: Could Not Recoganize Post Type

### ISSUE THREAD

**Issue by LoganAC34** [JackhammerYT/RedDownloader#27]
Hi. I'm having an issue. Everytime I try to download media from a post, all I get is an error.

My code:
```
url = www.Reddit.com/r/AnimalsBeingDerps/comments/en0ho3/excited_new_parent_playing_with_new_kittens/
RedDownloader.Download(url, output=submission.title, destination="media\\")
```

The error:

`Error: Could Not Recoganize Post Type`

Any help is appreciated. Thanks!

Side note: "Recoganize" is misspelled. It should be "Recognize".

**Comment by HVCFOGEY34** [comment]
So I figured out my issue with this. After digging around in the source files, I found an example link, and it looks like I was missing "https:\\ from my url.

Two suggestions after solving this:
It would be nice to show an example url in the readme of the repository.
It would be nice if we didn't need to include that or "www.reddit.com" in the name, and instead only need to include "/r/...". That way, it natively works with the PRAW command submission.permalink.

I also ran into another issue after solving this. I posted another issue here: https://github.com/JackhammerYT/RedDownloader/issues/30
